doc/doxygen/src/using-rust.md

```diff
@@ -122,7 +122,7 @@ and needs some patches applied:
 ```shell
 $ rustup install nightly-2019-12-05
 $ rustup component add --toolchain nightly-2019-12-05 rustfmt rustc-dev
-$ cargo +nightly-2019-12-05 install c2rust
+$ cargo +nightly-2019-12-05 install --locked c2rust
```

Comment:
hm, isn't this either-or? either install from crates.io (using this line), or from the checked-out path? and, older cargo versions don't support/honor "--locked" when installing from crates.io (vs. local path), IIRC.

@chrysn knows more.

Reply:
That's a weird error in the documentation I wrote -- that line shouldn't be there in the first place, the relevant install is 3 lines down where I fixed the --locked in https://github.com/RIOT-OS/RIOT/pull/17504/files

(The c2rust uploaded to crates.io is also not the one working, no clue what went wrong there precisely; that changed line should just go away)